sw/airborne/math/ActiveSetCtlAlloc/src/lib/chol_math.c

```diff
@@ -0,0 +1,110 @@
+#include "chol_math.h"
+#include <stdio.h>
+
+#include <string.h>
+#include <math.h>
+#include <stdint.h>
+
+void pprz_cholesky_float(num_t **out, num_t **in, num_t* inv_diag, int n)
```

Comment:
Why not using the function that is already in the math lib ? In general, why the different solvers are not at the root of the math folder if they are generic (cholesky, qr, ...) ?